[Button] Improve the disclosure API for button, allowing user to spec﻿﻿﻿ify the direction the disclosure icon faces (#2431)
Shopify/polaris-react: UNRELEASED.md

### Enhancements

> - Changed Button's `disclosure` prop to accept `"up"` or `"down"` as acceptable values, controlling the direction the disclosure caret faces ([#2431](https://github.com/Shopify/polaris-react/pull/2431))

Reviewer: ```suggestion
- Changed Button's `disclosure` prop to be `boolean | "up" | "down"` allowing to specify the direction the disclosure icon ([#2431](https://github.com/Shopify/polaris-react/pull/2431))
```

Reviewer: Not sure which I like best :)

### Bug fixes

> - Fixed an issue which caused HSL colors to not display in Edge ((#2418)[https://github.com/Shopify/polaris-react/pull/2418])

Reviewer: ```suggestion
- Fixed an issue which caused HSL colors to not display in Edge ([#2418](https://github.com/Shopify/polaris-react/pull/2418))
```
